src/forms/login-popup/components/AccountActivationMessage.jsx

```diff
@@ -63,12 +60,20 @@ const AccountActivationMessage = ({ messageType = null }) => {
       break;
   }
 
+  useEffect(() => {
+    if (alertRef.current) {
```

Comment:
We don't need to explicitly set focus on this. The screen reader will read the message when the alert appears.

Reply:
The focus goes to the close button not on Alert so we need to set the focus explicitly.